### DISCUSSION TITLE
[T] Symbol in front of filename?

### DISCUSSION THREAD

**Discussion by wiraki** [jesseduffield/lazygit#3966] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/6a7bc0f5-e992-4bc3-ab87-a1976599187e)

I don't think I have seen this before. What does [T] mean?

**Comment by stefanhaller** [comment, marked as answer]
This is simply the file icon for a toml file.

I personally don't like file icons, I think they add more noise than value, so I turned them off using
```yaml
gui:
    showFileIcons: false
```

**Reply by wiraki** [reply]
Thank you!! I didn't realize this was an actual Nerd symbol, I just looks like normal text. \*facepalm\*
